zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -274,6 +280,18 @@ impl NetworkUpgrade {
     ) -> Duration {
         NetworkUpgrade::current(network, height).averaging_window_timespan()
     }
+
+    /// Returns true if the maximum block time rule is active for `network` and `height`.
+    ///
+    /// Always returns true if `network` is the Mainnet.
+    /// If `network` is the Testnet, the `height` should be at least
+    /// TESTNET_MAX_TIME_START_HEIGHT to return true.
+    /// Returns false otherwise.
+    ///
+    /// Part of the consensus rules at https://zips.z.cash/protocol/protocol.pdf#blockheader
+    pub fn is_max_block_time_enforced(network: Network, height: block::Height) -> bool {
+        network == Network::Mainnet || height >= TESTNET_MAX_TIME_START_HEIGHT
```

Comment:
I'm just going to refactor this code into an exhaustive match.

We plan on changing the Network enum in future, and exhaustive matches will make sure we update this code when we do.
```suggestion
        match network {
            Network::Mainnet => true,
            Network::Testnet => height >= TESTNET_MAX_TIME_START_HEIGHT,
        }
```